src/components/Charts/utils.ts

```diff
@@ -37,13 +37,14 @@ const parseBar = (framesArray: SingleFramesLoose[], key: string, samplePeriod: n
     if (frame[importance]) importantIndexes.push(i);
     return { id: i, Time: i * samplePeriod, [goodKeyName]: Math.round(Number(frame[key]) * 10 * 10) / 10 };
   });
-
   return { data, importantIndexes, keys: [goodKeyName] };
 };
 
+// TODO when we get links for old videoanalysis from be, add if else statement if we receive Uint8Array or another type in importantFrame property
 export const getImportantFrames = (data: SingleFrameAnalysis[]) => {
   const frames: Frame = {};
-  data.forEach((singleFrameData, i) => singleFrameData.importantFrame && (frames[i] = singleFrameData.importantFrame));
+  data.forEach((singleFrameData, i) => singleFrameData.importantFrame &&
+    (frames[i] = URL.createObjectURL(new Blob([singleFrameData.importantFrame], { type: 'image/jpg' }))));
```

Comment:
Moved the logic for creating links to here, I understand this is what you were mentioning to me yesterday, nice

Reply:
Precisely!